internal/controller/bucket/helpers.go

```diff
@@ -109,21 +109,41 @@ func setBackendLabels(bucket *v1alpha1.Bucket, providerNames []string) {
 	}
 }
 
-func setBucketStatus(bucket *v1alpha1.Bucket, bucketBackends *bucketBackends, providerNames []string) {
+func getBucketProvidersFilterDisabledLabel(bucket *v1alpha1.Bucket, providerNames []string) []string {
```

Comment:
There was a comment above this function explaining it before it was moved over from the utils package. I think it would be useful to keep.